src/rcpp_octave.cpp

```diff
@@ -59,15 +62,18 @@ extern "C" {
 #endif
 #include <octave/error.h>
 #include <octave/quit.h>
+#include <getopt.h>
+
 #if SWIG_OCTAVE_PREREQ(4,1,0) // version >= 4.1.0
 // Must handles issue #14 here
 // see: http://octave.org/doxygen/4.1/da/d0d/signal-wrappers_8h.html
-//#include <octave/signal-wrappers.h>
+#include "signal-wrappers.h"
 #endif
 #include <octave/variables.h>
 #include <octave/sighandlers.h>
 #include <octave/sysdep.h>
-#include <octave/str-vec.h>
+
+#include <ov-usr-fcn.h>
```

Comment:
I just did `hgview -n ./libinterp/octave-value/ov-usr-fcn.h` on the octave repo. The file was added back in 1997 and should be safe to include for any version of octave.